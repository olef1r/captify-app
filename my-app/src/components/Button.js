import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Button from '@material-ui/core/Button';
import Container from '@material-ui/core/Container';

const useStyles = makeStyles(theme => ({
  button: {
    marginBottom: '-80px',
    marginLeft: '34rem'
  },
  text: {
    display: 'block',
    marginTop: '102px',
    marginLeft: '27rem',
    color: 'red'
  }
}));

export default function OutlinedButtons({ onClick, disabled }) {
  const classes = useStyles();
  return (
    <Container maxWidth='xs'>
      <Button variant="outlined" 
        onClick={onClick}
        disabled={disabled}
        className={classes.button}>
        Click
      </Button>
      {disabled ? <p className={classes.text}>You can add more items</p> : null}
    </Container>
  );
}