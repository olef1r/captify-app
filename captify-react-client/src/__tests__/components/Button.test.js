import React from 'react';
import { shallow } from 'enzyme';
import Button from '../../components/Button';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Button/>);
});

describe('<Button> - Component', () => {
  it('renders without crashing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
