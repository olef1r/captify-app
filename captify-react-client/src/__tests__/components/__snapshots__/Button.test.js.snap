// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button> - Component renders without crashing 1`] = `
<WithStyles(ForwardRef(Container))
  maxWidth="xs"
>
  <WithStyles(ForwardRef(Button))
    className="makeStyles-button-1"
    variant="outlined"
  >
    Click
  </WithStyles(ForwardRef(Button))>
</WithStyles(ForwardRef(Container))>
`;
